python/tank/util/qt_importer.py

```diff
@@ -111,7 +125,7 @@ def _import_pyside(self):
 
         :returns: The (binding name, binding version, modules) tuple.
         """
-        from PySide import QtCore, QtGui
+        from PySide import QtCore, QtGui, QtWebKit, QtNetwork
```

Comment:
I created a new method (_import_module_by_name) which does load QtWebKit and QtNetwork in a way where the app will not crash if they are not present.